core/src/main/scala/water/fvec/H2OFrameUtils.scala

```diff
@@ -0,0 +1,27 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package water.fvec
+
+
+/**
+ * This is a simple bridge to access package-private/protected members.
+ */
+object H2OFrameUtils {
```

Comment:
Method `getChunksLocations` was unused and all other are already defined on ChunkUtils except the `createNewChunks` with `sparse` argument. Created PR on H2O side and SW side we can remove this duplication